packages/forms/src/model.ts

```diff
@@ -1450,22 +1450,36 @@ export class FormGroup extends AbstractControl {
    *
    * @param name The control name to add to the collection
    * @param control Provides the control for the given name
+   * @param options Allows to specify whether this FormGroup instance should emit events after a new
```

Comment:
Docs NIT: is `Allows to` necessary?

```suggestion
   * @param options Specify whether this FormGroup instance should emit events after a new
```